pdns/ssqlite3.cc

```diff
@@ -234,19 +234,28 @@ std::unique_ptr<SSqlStatement> SSQLite3::prepare(const string& query, int nparam
 
 void SSQLite3::execute(const string& query) {
   char *errmsg;
-  int rc;
-  if (sqlite3_exec(m_pDB, query.c_str(), NULL, NULL, &errmsg) == SQLITE_BUSY) {
+  std::string errstr1;
+  int rc = sqlite3_exec(m_pDB, query.c_str(), NULL, NULL, &errmsg);
```

Comment:
Not your doing but since we are refactoring that code anyway:
```suggestion
  int rc = sqlite3_exec(m_pDB, query.c_str(), nullptr, nullptr, &errmsg);
```